### PR TITLE
Review `Code actions` and `Debug` pages

### PR DESCRIPTION
**Pull request by praneesha**
Review `Code actions` and `Debug` pages

### CODE REVIEW

File: en/docs/debug.md [hunk 1]
```diff
@@ -98,37 +96,25 @@ Follow the steps below to start a remote debug session.
 
 3. After setting the remote debug configurations, select **Ballerina Remote** from the drop-down available in the upper left corner to start a remote debugging session.
 
-4. Open a terminal and execute the debug command that is relevant to your requirement:
+4. Open a terminal and execute the debug command that is relevant to your requirement from the ones below:
 
-   -  Debugging a Ballerina package or a single file:
+      | Code action                     	| Description                                                          	|
+      |---------------------------------	|----------------------------------------------------------------------	|
+      | Debugging a Ballerina package or a single file:                 	| `bal run --debug <DEBUGGEE_PORT> <BAL_FILE_PATH/PACKAGE_PATH>`
```
Comment (nilminiwso2) [inline en/docs/debug.md:103]:
```suggestion
      | Debugging a Ballerina package or a single file                 	| `bal run --debug <DEBUGGEE_PORT> <BAL_FILE_PATH/PACKAGE_PATH>`
```

File: en/docs/debug.md [hunk 2]
```diff
@@ -98,37 +96,25 @@ Follow the steps below to start a remote debug session.
...
+      | Debugging a Ballerina executable JAR:                	| `bal run --debug <DEBUGGEE_PORT> <EXECUTABLE_JAR_FILE_PATH>`|
```
Comment (nilminiwso2) [inline en/docs/debug.md:104]:
```suggestion
      | Debugging a Ballerina executable JAR                	| `bal run --debug <DEBUGGEE_PORT> <EXECUTABLE_JAR_FILE_PATH>`|
```

File: en/docs/debug.md [hunk 3]
```diff
@@ -164,58 +146,39 @@ Each configuration supports a different set of attributes, which will be provide
 
 The following attributes are mandatory for all configurations.
 
-- `name` - The reader-friendly name to appear in the Debug launch configuration drop-down menu.
-- `type` - The type of debugger to use for this launch configuration. The attribute value must be kept as `ballerina` for all Ballerina debugging configuration types.
-- `request` - The request type of this launch configuration. Currently, `launch` and `attach` are supported.
+| Attribute                     	| Description                                                          	|
+|---------------------------------	|----------------------------------------------------------------------	|
+| `name`                 	| The reader-friendly name to appear in the Debug launch configuration drop-down menu                                  	|
```
Comment (nilminiwso2) [inline en/docs/debug.md:151]: any reason why we dropped the full stop?

File: en/docs/debug.md [hunk 4]
```diff
@@ -46,20 +44,20 @@ Follow the steps below to start a debug session with configurations. All the con
 
 2. Press the **Control + Shift + D** keys (for Mac: **Command + Shift +D**) to launch the Debugger view.
 
-3. Click **create a launch.json** file and then select **Ballerina Debug** as the **Environment**.
+3. Click **create a launch.json** file and then select **Ballerina Debug** as the **Environment**. 
 
-   You can view the opened `launch.json` file.
+      You view the opened `launch.json` file.
```
Comment (nilminiwso2) [inline en/docs/debug.md:49]: I think `You can` is more clear.
Or better - `You will see the...`

but not an error.

File: en/docs/debug.md (hunk 3 above)
Comment (praneesha) [reply en/docs/debug.md:151]: Do we need to use full stops in bullet points?

File: en/docs/debug.md (hunk 4 above)
Comment (praneesha) [reply en/docs/debug.md:49]: isn't the best practice to use direct voice in technical documentation?

Comment (nilminiwso2) [reply en/docs/debug.md:49]: We shouldn't be using `will`, yes. Need to change that in multiple places.

`You view.. ` in this sentence seems very odd IMO. But let's proceed with your text.

File: en/docs/debug.md (hunk 3 above)
Comment (nilminiwso2) [reply en/docs/debug.md:151]: This is a table right?
In bullet lists, if it is a complete sentence -yes

File: en/docs/debug.md [hunk 5]
```diff
@@ -164,58 +146,39 @@ Each configuration supports a different set of attributes, which will be provide
 
 The following attributes are mandatory for all configurations.
 
-- `name` - The reader-friendly name to appear in the Debug launch configuration drop-down menu.
-- `type` - The type of debugger to use for this launch configuration. The attribute value must be kept as `ballerina` for all Ballerina debugging configuration types.
-- `request` - The request type of this launch configuration. Currently, `launch` and `attach` are supported.
+| Attribute                     	| Description                                                          	|
+|---------------------------------	|----------------------------------------------------------------------	|
+| `name`                 	| The reader-friendly name to appear in the Debug launch configuration drop-down menu                                  	|
+| `type`  	| The type of debugger to use for this launch configuration. The attribute value must be kept as `ballerina` for all Ballerina debugging configuration types                   	|
```
Comment (nilminiwso2) [inline en/docs/debug.md:152]: for example here - the first sentence in the description has a full stop and the second one doesn't.

File: en/docs/debug.md (hunk 4 above)
Comment (anupama-pathirage) [reply en/docs/debug.md:49]: Yes, I agreed to @nilminiwso2's point here. `You view the ...` seems incomplete or odd.

Comment (praneesha) [reply en/docs/debug.md:49]: Changed to `You will see the...`.

File: en/docs/debug.md (hunk 5 above)
Comment (praneesha) [reply en/docs/debug.md:152]: Added the full stops back.

File: en/docs/edit-the-code/code-actions.md [hunk 6]
```diff
@@ -47,106 +47,120 @@ The following quick fixes are available for check expressions.
 
 | Code action                     	| Description                                                          	|
 |---------------------------------	|----------------------------------------------------------------------	|
-| `Surround with do/on-fail`                 	| Surround a check expression with a do/on-fail clause                                  	|
-| `Change return type to 'error?'`  	| Change the return type of a function/method to `error?`                   	|
+| `Surround with do/on-fail`                 	| Surround a check expression with a do/on-fail. clause                                  	|
```
Comment (nilminiwso2) [inline en/docs/edit-the-code/code-actions.md:50]:
```suggestion
| `Surround with do/on-fail`                 	| Surround a check expression with a do/on-fail clause.                                  	|
```

File: en/docs/edit-the-code/code-actions.md [hunk 7]
```diff
@@ -179,32 +197,40 @@ Source actions are used to generate common code structures and recurring element
...
+<img src="https://github.com/wso2/docs-ballerina-visual-studio-code-plugin/blob/main/en/docs/img/source-actions-for-getters-and-setters.gif?raw=true" width="100%" />
 
 ### Variables
 
 The following source actions are available for variables.
 
-- `Type guard variable`: Type guard a variable if the variable type is a union type.
+| Code action                     	| Description                                                          	|
+|---------------------------------	|----------------------------------------------------------------------	|
+| `Type guard variable`                	| Type guard a variable if the variable type is a union. type                     	|
```
Comment (nilminiwso2) [inline en/docs/edit-the-code/code-actions.md:214]:
```suggestion
| `Type guard variable`                	| Type guard a variable if the variable type is a union type.                     	|
```

**Review by nilminiwso2 — APPROVED** [review]

**Pull request merged**